<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>标定检测</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1111</width>
      <height>961</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>标定误差</string>
     </attribute>
     <widget class="QPushButton" name="draw_right">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>400</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>单目误差/右</string>
      </property>
     </widget>
     <widget class="QPushButton" name="draw_multi">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>双目误差</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>120</y>
        <width>431</width>
        <height>371</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="figLayout1"/>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>431</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="path_btn">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>文件夹选择</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>40</y>
        <width>261</width>
        <height>851</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="draw_left">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>310</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>单目误差/左</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>520</y>
        <width>621</width>
        <height>371</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="figLayout2"/>
     </widget>
     <widget class="QCheckBox" name="check_outer">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>260</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>仅外参</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_model">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>生成标定模型</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>测试</string>
     </attribute>
     <widget class="QLineEdit" name="line2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>431</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="path2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>文件夹选择</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>130</y>
        <width>431</width>
        <height>371</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="figLayout3"/>
     </widget>
     <widget class="QTextBrowser" name="textBrowser2">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>40</y>
        <width>261</width>
        <height>851</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>520</y>
        <width>621</width>
        <height>371</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="figLayout4"/>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>180</y>
        <width>102</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="draw_multi2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>双目误差</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_outer2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>340</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>仅外参</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1089</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>path_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>mybutton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>draw_multi</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>draw_cmd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>draw_left</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>draw_cmd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>draw_right</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>draw_cmd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>577</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>path2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>mybutton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>544</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>draw_multi2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>draw_cmd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>544</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>mybutton()</slot>
  <slot>draw_cmd()</slot>
  <slot>reset_cmd()</slot>
 </slots>
</ui>
